.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 # For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
 
-name: Test | Build
```

Comment:
Why?

Reply:
To match `build.yml`.

Reply:
haha but we are Testing and Linting!  If anything, it should be `Lint | Test | Build`